securedrop/source_templates/index.html

```diff
@@ -54,12 +76,14 @@
     Included here so the images can preload while the user is first
     reading the page. Hidden by default. -->
     <div class="bubble">
-      <p>You appear to be using the Tor Browser. You can disable Javascript in 3 easy steps!</p> 
+      <p>You appear to be using the Tor Browser. You can turn the Security Slider to High in 4 easy steps!</p>
       <ol>
-        <li>Click the <img src="static/i/no16.png" alt="NoScript icon" /> NoScript icon in the toolbar above</li>
-        <li>Click <strong><img src="static/i/no16-global.png" alt="Forbid Scripts Globally" /> Forbid Scripts Globally (advised)</strong></li>
-        <li>If the page does not refresh automatically, <a href="/">click here</a> to refresh the page</li> 
+        <li>Click the <img src="{{ url_for('static', filename='i/toronion.png') }}" alt="Tor icon" />Tor icon in the toolbar above</li>
+        <li>Click <strong>Security Settings</strong></li>
+        <li>Turn the Slider to <strong>High</strong></li>
+        <li>If the page does not refresh automatically, <a href="/">click here</a> to refresh the page</li>
       </ol>
+
       <p><a href="/howto-disable-js">Not using the Tor Browser Bundle?</a></p>
```

Comment:
While we're here, you should go ahead and remove this pointless and confusing link. As we discussed in the office:

1. The page it links to just provides instructions for Tor Browser again. `/howto-disable-js` *used* to have instructions for disabling JS in other browsers, but at some point people argued that sources really shouldn't be using a non-Tor Browser browser, so those instructions were removed but this link stayed. It is confusing to click "Not using the Tor Browser Bundle?," which implies you will be taken to instructions for other browsers, only to get the Tor Browser instructions again in a slightly different form.
2. This pop-up only appears if you're using Tor Browser (see `is_likely_tor_browser` in `source.js`), so this link is just generally kind of pointless.